recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,48 @@
+python_version:
+# 2020.09
+  - 3.6.*
+  - 3.7.*
+  - 3.8.*
+  - 3.9.*
+# 2021.03
+  - 3.6.*
+  - 3.7.*
+  - 3.8.*
+  - 3.9.*
+# 2021.09
+  - 3.7.*
+  - 3.8.*
+  - 3.9.*
+# 2022.03
+  - 3.7.*
+  - 3.8.*
+  - 3.9.*
+  - 3.10.*
+# 2022.09
+  - 3.8.*
+  - 3.9.*
+  - 3.10.*
+  - 3.11.*
 rdkit_version:
-  - 2021.09.*
-  - 2021.03.*
   - 2020.09.*
+  - 2020.09.*
+  - 2020.09.*
+  - 2020.09.*
+  - 2021.03.*
+  - 2021.03.*
+  - 2021.03.*
+  - 2021.03.*
+  - 2021.09.*
+  - 2021.09.*
+  - 2021.09.*
+  - 2022.03.*
+  - 2022.03.*
+  - 2022.03.*
+  - 2022.03.*
+  - 2022.09.*
+  - 2022.09.*
+  - 2022.09.*
+  - 2022.09.*
+zip_keys:
+  - python_version
+  - rdkit_version
```

Comment:
I don't think it's needed to enumerate the python versions and we should stick with conda-forge defaults.

Can you replace by:

```yaml
rdkit_version:
  - 2022.09.*
  - 2022.03.*
  - 2021.09.*
```

?

Ideally we should try to minimize the number of rdkit versions and only include the last 2-3.

Reply:
You'll need to rerender ofc after that.

Reply:
If some combinations are not desired, we will exclude them later in the meta recipe file.